storage/src/main/java/org/apache/kafka/server/log/remote/storage/RemoteLogManagerConfig.java

```diff
@@ -81,6 +81,13 @@ public final class RemoteLogManagerConfig {
     public static final String REMOTE_LOG_METADATA_MANAGER_LISTENER_NAME_DOC = "Listener name of the local broker to which it should get connected if " +
             "needed by RemoteLogMetadataManager implementation.";
 
+    public static final String REMOTE_LOG_METADATA_CUSTOM_METADATA_MAX_SIZE_PROP = "remote.log.metadata.custom.metadata.max.size";
```

Comment:
Can `remote.log.metadata.custom.metadata.max.size` be changed to `remote.log.metadata.custom.metadata.max.bytes` as we follow similar convention in other places? 

We need to update the KIP with this change and reply to the vote thread about this change if other reviewers are fine with this change. 
cc @divijvaidya @showuon @kamalcph

Reply:
No concerns from my side.

Reply:
Sure. Done this and updated the KIP.

Reply:
No concern. Thanks.